solutions/functions/prime-checker-function.js

```diff
@@ -0,0 +1,30 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+// Following function takes integer and should return True if it's prime 
+// otherwise  should return False.
+function is_prime(input_number){
```

Comment:
In the template, the functions  were in camelCase, but this solution uses snake_case for representing functions and variables. As per the coding conventions mentioned, we only use camelCase for functions and variables and PascalCase for classes.

Reply:
I have updated this template code

Reply:
Okay updated the code as per the latest template.